options/glibc/generic/shadow-stubs.cpp

```diff
@@ -17,6 +24,150 @@ int putspent(const struct spwd *sp, FILE *f) {
 }
 #undef NUM
 
+static long xatol(char **s) {
+	long x;
+	if(**s == ':' || **s == '\n') {
+		return -1;
+	}
+	for(x = 0; **s - '0' < 10U; ++*s) {
+		x = 10 * x + (**s - '0');
+	}
+	return x;
+}
+
+int __parsespent(char *s, struct spwd *sp) {
```

Comment:
Is this supposed to be a public function?

Reply:
Whoops, certainly not. As far as I can tell this should be internal. Good catch.

Reply:
Fixed